src/adaPets.js

```diff
@@ -5,33 +5,59 @@ const result = require('./result.js');
 const setResult = result.setResult;
 const setError = result.setError;
 
-const BASE_URL = 'http://localhost:3000/pets/';
+const BASE_URL = 'http://localhost:3000/pets';
 
 // Option functions.
 const listPets = () => {
-  // Fill out as part of Wave 1.
+  axios.get(BASE_URL)
+  .then(function (response) {
+    setResult(response.data);
+  })
+  .catch(function (error) {
+    setError("Unable to retrieve pets data. Sorry!");
+    console.log(error.response.status);
+  })
 };
 
 const showDetails = (selectedPetId) => {
   if (!selectedPetId) {
     setError("You tried to show details for a pet without selecting it!");
-    
   }
 
   // Fill out as part of Wave 2.
+  axios.get("http://localhost:3000/pets/" + selectedPetId)
```

Comment:
You should have used `BASE_URL` here:

```suggestion
  axios.get(BASE_URL + selectedPetId)
```